mediawiki123-apb/roles/provision-mediawiki123-apb/tasks/main.yml

```diff
@@ -55,16 +55,6 @@
         value: postgres
       - name: MEDIAWIKI_SITE_SERVER
         value: '{{ route.route.spec.host }}'
-      - name: POSTGRESQL_HOST
-        value: '{{ postgresql_host }}'
-      - name: POSTGRESQL_DATABASE
-        value: '{{ postgresql_database }}'
-      - name: POSTGRESQL_PORT
-        value: '{{ postgresql_port }}'
-      - name: POSTGRESQL_USER
-        value: '{{ postgresql_user }}'
-      - name: POSTGRESQL_PASSWORD
-        value: '{{ postgresql_password }}'
       image: docker.io/jmontleon/mediawiki123:latest
```

Comment:
In as many cases as possible we're reusing existing containers. So we're pointing where ever existing working containers are. We didn't have a suitable mediawiki container (as this is intended to be brought downstream an instance installed via RPM was necessary) so we made our own.

Here, we could make the case it could be under ansibleplaybookbundles. I don't know if we only want to use the org for APB's? At any rate it was just a decision when I created the container to push it to my personal account.